### PR TITLE
Enhance of MyBaseMVController

### PR DESCRIPTION
**Pull request by daisukenishino**
- Enhance of MyBaseMVController  
https://github.com/OpenTouryoProject/OpenTouryo/issues/118
- "404 Not Found" occurs when I run "MVC_Sample". https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/16
- An exception occurs in the redirection method in child action. https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/17

### CODE REVIEW

**Comment by daisukenishino** [comment]
@kishoreandukuri 

Is the test of this fix performed enough ?

- Session time-out function.
- Error Screen transition at the child action by the Html.Action() of when an exception occurs.  
"Child view's Controller that inherited the MyBaseMVController can be called using the HTML.Action method from main controller that has not directly accessed by URL."
- Output of log from the MyBaseMVController.
- "404 Not Found" does not occur at MVC_Sample.

**Comment by kishoreandukuri** [comment]
@daisukenishino 

we have tested enough from our side

**Comment by daisukenishino** [comment]
@yusukemaegawa 
Please do the acceptance confirmation at the SymphonyTeleca/OpenTouryoTemplates repository's nishino branch that is merged with the [pullrequest#22](https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/22), And merge this pullrequest.

File: root_VS2012/programs/C#/Frameworks/Infrastructure/Business/Business.csproj [hunk 1]
```diff
@@ -89,6 +89,7 @@
     <Compile Include="Exceptions\MyBusinessApplicationExceptionMessage.cs" />
     <Compile Include="Exceptions\MyBusinessSystemExceptionMessage.cs" />
     <Compile Include="Presentation\CmnTableAdapter.cs" />
+    <Compile Include="Presentation\BaseMVController.cs" />
```
Comment (yusukemaegawa) [inline root_VS2012/programs/C#/Frameworks/Infrastructure/Business/Business.csproj]: @daisukenishino

Please include this class in **Framework.csproj**.

- BaseMVController.cs
    - You should prohibit the user from customizing.
    - You should include this class in Framework.csproj.
- MyBaseController.cs
    - You should allow the user to customize.
    - You should include this class in Business.csproj.

**Comment by yusukemaegawa** [comment]
@daisukenishino 

Project Configuration is wrong.
Set [comment](https://github.com/OpenTouryoProject/OpenTouryoTemplates/pull/21#discussion_r44501237).

Please correct this error and re-send Pull Request.

**Pull request closed without merging**